HW02_B_Mimic.md

```diff
@@ -1,89 +1,151 @@
+================
+Ariana Tortolani
+
+``` r
+library("ggplot2")
+library("magrittr") #so I can do some piping
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library("ggrepel")
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I
+make from scratch. I will only provide the MD, not the actual code I
+used to create it besides which data I use to create it. The rest will
+be up to you.
+
+Try for all 4, but if you are struggling don’t worry about it. Try your
+best for each, if you don’t get everything that’s what the peer-review
+is for. :smile:
+
+### Graph 1
+
+``` r
+data("diamonds")
+#hint think about the *position* the bars are in...
+```
+
+Using the diamonds dataset, make this graph:
+![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-2-1.png)<!-- -->
+
+``` r 
+ggplot(diamonds, aes(cut, fill = clarity)) +
+  geom_bar(position = position_dodge()) +
+  labs(title = "My Diamond Collection", 
+       subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond") +
+  theme(plot.title = element_text(size = 14, face = "bold", hjust = 0.5)) +
+  labs (x = "Diamond Cut", y = "Number of Diamonds") +
+  annotate(geom = "text", x = 4, y = 4500, label = "My Best Diamonds,\nof course") +
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, alpha = 0.25)
+
+```
+
+My Graph:
+![MY GRAPH](HW02_B_Mimic_starter_files/figure-gfm/graph1 code-1.png)<!-- -->
+
+### Graph 2
+
+``` r
+data("iris")
+```
+
+Using the iris dataset, make this graph:
+
+    ## `geom_smooth()` using formula 'y ~ x'
+
+![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-4-1.png)<!-- -->
+
+```r 
+
+iris$NewOrder <- factor(iris$Species, levels = c("versicolor","setosa","virginica"))
+
+ggplot(iris, aes(Sepal.Length, Petal.Length, color = Species, shape = Species)) +
+  geom_point() +
+  geom_smooth(method = lm, se = FALSE, color = "black") +
+  facet_wrap(vars(NewOrder), scales = "free_y") 
+
+```
+
+My Graph:
+![MY GRAPH](HW02_B_Mimic_starter_files/figure-gfm/graph 2 code-1.png)<!-- -->
+
+### Graph 3
+
+You’ll need the information in this first box to create the graph
+
+``` r
+data("mpg")
+corvette <- mpg[mpg$model == "corvette",]
+#install
+require("ggrepel") #useful for making text annotations better, hint hint
+set.seed(42)
+```
+
+Now using the mpg dataset and the corvette dataset, make this graph:
+
+![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-6-1.png)<!-- -->
+
+```r
+
+ggplot(mpg, aes(displ,hwy)) +
+  geom_point() + 
+  geom_point(data = corvette, color = "blue") +
+  geom_text_repel(data = corvette, aes(label = paste("Corvette,",year))) +
+  scale_x_continuous(limits = c(1,8), breaks = seq(1,8,1)) +
+  labs(title = "Corvettes are a bit of an outlier") 
```

Comment:
I noticed on your last blue point on the graph, there is a line that is attached to the label (Corvette, 2008). I had a similar problem - I ended up concluding that the problem was caused by the lack of space for the label to be appropriately positioned such that it appears directly associated with the point and fixed it by minimizing the font size. I am not sure if that is the best solution!